independent-publisher-connectors/AccuWeather/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "AccuWeather API Key",
+          "description": "The app/subscription API key issued by AccuWeather",
+          "tooltip": "Provide API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FF4400",
```

Comment:
@ahmadn82 Please update it with `"#da3b01"` as it is the default iconBrandColor for all the Independent Publisher connectors.

Reply:
Fixed! @sriyen-msft